ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -266,6 +266,24 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
           )
     }
 
+    @Explanation(
+      "This error is emitted, when a submitted ledger API command specifies a deduplication period exceeding the maximum."
+    )
+    @Resolution(
+      "Adjust the value of the deduplication period or ask the participant operator to increase the maximum."
+    )
```

Comment:
```suggestion
    @Explanation(
      "This error is emitted when a submitted ledger API command specifies an invalid deduplication period."
    )
    @Resolution(
      "Inspect the error message, adjust the value of the deduplication period or ask the participant operator to increase the maximum."
    )
```
This explanation is more specialized than the actual error definition. Let's adjust